internal/backend/azure/azure.go

```diff
@@ -124,12 +126,25 @@ func open(cfg Config, rt http.RoundTripper) (*Backend, error) {
 		}
 	}
 
+	validAccessTiers := blob.PossibleAccessTierValues()
+	var accessTier blob.AccessTier
+
+	// if the access tier is not valid, then we will not set the access tier; during the upload process,
+	// the value will be inferred from the default configured on the storage account.
+	for _, tier := range validAccessTiers {
+		if strings.Compare(strings.ToLower(string(tier)), strings.ToLower(cfg.AccessTier)) == 0 {
+			accessTier = tier
+			break
+		}
+	}
```

Comment:
@MichaelEischer check here, about the handling of valid access tiers. I have done a case-insensitve check after validating the behaviour with Azure APIs and live debugging.